docs/src/main/asciidoc/spanner.adoc

```diff
@@ -1435,10 +1437,11 @@ This command can be used to create Cloud Spanner instances:
 $ gcloud spanner instances create <instance-name> --config=emulator-config --description="<description>" --nodes=1
 ----
 
-Remember that you need to set the `SPANNER_EMULATOR_HOST` environment variable to use the emulator:
```

Comment:
Not sure if we want to remove this. 
Setting/unsetting the variable is a quick way to switch all your tests to using the emulator and back to normal. Might be still useful.

Reply:
I'm fine with removing it. Even though it's available, it's not consistent with the other emulators. We can leave a note about it, but I'm afraid it will only confuse people.